src/evm_context.rs

```diff
@@ -132,6 +141,33 @@ pub fn calldataload(index: &String, _ssa: &mut SSATracker) -> String {
     format!("(concat {})", arguments)
 }
 
+pub fn mstore(index: &String, value: &String, ssa: &mut SSATracker) -> String {
+    let before = ssa.to_smt_variable(&memory());
+    let after = ssa.allocate_new_ssa_index(&memory());
+
+    let stored = (0..32).fold(before.name, |acc, i| {
+        let sub_index = format!("(bvadd {} #x{:064X})", index, i);
+        let byte_value = format!(
+            "((_ extract {} {}) {})",
```

Comment:
Why do you need to extract? Aren't the memory elements BV8 anyway?

Reply:
No, this is the value to be stored, it's a regular stack word.

Reply:
ah true